tests/profiling/exporter/test_pprof.py

```diff
@@ -346,6 +349,8 @@
             thread_native_id=123987,
             thread_name="MainThread",
             trace_id=1322219321,
+            trace_resource="notme",
+            trace_type="sql",
```

Comment:
Is this and the one below intentional or should have been `SpanTypes.SQL.value`?

Reply:
Whatever. I mean both works and are valid here.
What we want is that if it's not `SpanTypes.WEB` we don't store any trace_resource, so it being the "SQL" span type or the customer `sql`, it must be the same behaviour :)